src/components/prompts/FeedbackForm.js

```diff
@@ -220,7 +220,7 @@ class FeedbackPrompt extends React.Component {
           </Button>
           <Button
             onClick={ this.onReady }
-            color='teal'>Ready
+            color='teal'>Finished
```

Comment:
I'm neutral on this, but you would probably have to ask a non-native speaker if one is really easier than the other. I think it's clever to try to use a cognate! But "ready" is also a pretty simple word, so in reality they might be equivalent for most people who know any English.

Aside from that question though, "Finished" probably makes more sense here anyway, since "Ready" kind of implies that there's something that comes next, rather than just being "finished" with the form.

Reply:
There is something that comes next - that question about sending data or not. I agree that what we really need is some user testing. I wonder if we could do that kind of testing more easily than the other? On mechanical turk somehow or something?